Lombiq.BaseTheme.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -43,4 +43,11 @@ public static async Task TestMainMenuWithAuthenticationAsync(this UITestContext
         await context.ClickMainMenuPathAsync("Log In");
         context.Exists(By.XPath("//form[@action = '/Login']/*[starts-with(name(), 'h') and contains(., 'Log in')]"));
     }
+
+    public static async Task BaseThemeDependencyShouldBeEnabledAsync(this UITestContext context)
+    {
+        await context.GoToAdminRelativeUrlAsync("/Features");
+        await context.ClickAndFillInWithRetriesAsync(By.Id("search-box"), "Helpful Widgets");
```

Comment:
You don't need this before the existence check in the next line, that should work on its own too.